zk_toolbox/crates/zk_inception/src/commands/ecosystem/init.rs

```diff
@@ -94,42 +91,18 @@ pub async fn run(args: EcosystemInitArgs, shell: &Shell) -> anyhow::Result<()> {
         .await?;
     }
 
-    // If the name of chain passed then we deploy exactly this chain otherwise deploy all chains
-    let list_of_chains = if let Some(name) = global_config().chain_name.clone() {
-        vec![name]
-    } else {
-        ecosystem_config.list_of_chains()
-    };
-
-    for chain_name in &list_of_chains {
-        logger::info(msg_initializing_chain(chain_name));
-        let chain_config = ecosystem_config
-            .load_chain(Some(chain_name.clone()))
-            .context(MSG_CHAIN_NOT_INITIALIZED)?;
-
-        let mut chain_init_args = chain::args::init::InitArgsFinal {
-            forge_args: final_ecosystem_args.forge_args.clone(),
-            genesis_args: genesis_args.clone().fill_values_with_prompt(&chain_config),
-            deploy_paymaster: final_ecosystem_args.deploy_paymaster,
-            l1_rpc_url: final_ecosystem_args.ecosystem.l1_rpc_url.clone(),
-            port_offset: PortOffset::from_chain_id(chain_config.id as u16).into(),
-        };
-
-        chain::init::init(
-            &mut chain_init_args,
-            shell,
-            &ecosystem_config,
-            &chain_config,
-        )
-        .await?;
+    // Initialize chain(s)
+    let mut chains: Vec<String> = vec![];
```

Comment:
it's weird decision. why do we need this list?

Reply:
Just for logging: to let user know what chains were initialized in the end. It was there before and I thought why not keeping that.
I can't just print `ecosystem_config.list_of_chains()` since it's unknown if we're initializing one chain of several.